app/presenters/mfa_confirmation_presenter.rb

```diff
@@ -1,6 +1,9 @@
 class MfaConfirmationPresenter
-  def initialize(user)
-    @user = user
+  attr_reader :mfa_context
+
+  def initialize(mfa_context:, show_skip_additonal_mfa_link: true)
+    @mfa_context = mfa_context
```

Comment:
It doesn't appear we use the MFA context on the presenter? Should it be removed?

Alternatively, we could move the logic for `show_skip_additonal_mfa_link?` into the presenter?

Reply:
There's a trade off between defining the logic of the function `show_skip_additional_mfa_link?` once or duplicating the logic in the following two files:

`app/presenters/mfa_confirmation_presenter.rb`
`app/presenters/two_factor_options_presenter.rb`

Presently the logic is defined in one location (`controllers/concerns/mfa_setup_concern.rb`). I'm not-so-attached to either style.

The unnecessary mfa_context has been removed and was left over from a change in style.